contracts/allocs/scripts/genallocs.go

```diff
@@ -105,7 +105,7 @@ func allocConfig(network netconf.ID, prefunds types.GenesisAlloc) (bindings.Allo
 	}
 
 	return bindings.AllocPredeploysConfig{
-		Admin:                  eoa.MustAddress(network, eoa.RoleAdmin),
+		Admin:                  eoa.MustAddress(network, eoa.RoleManager),
```

Comment:
this sets the manager as owner of all predeployed I think, while we want to some to be owned by the upgrader. Like octane predeployes. Maybe even all predeployes, I'm not sure about OmniNativeBridge.

Reply:
Yes we need to have a split.  Which means updating forge script. I can handle that on add to this PR

Reply:
I'm doing it no worries

Reply:
I will just need you to likely correct me after @kevinhalliday :D